spec/jobs/decidim/idcat_mobil/verification_job_spec.rb

```diff
@@ -29,9 +29,7 @@ def pending_to_be_finished
     # rubocop: disable Lint/ConstantDefinitionInBlock
     class TestDecidimPublisher < Decidim::Command
       include Wisper::Publisher
-      # rubocop: disable Lint/MissingSuper
       def initialize(*args); end
```

Comment:
we don't want super?
```suggestion
      def initialize(*args)
        super
      end
```